relayer/src/channel.rs

```diff
@@ -224,18 +222,20 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
                 connection_id,
                 port_id,
                 channel_id,
+                // The event does not include the version.
+                // The message handlers `build_chan_open..` determine the version from channel query.
+                // TODO(Adi): Investigate proto correctness wrt version.
```

Comment:
Reminder for this TODO

Reply:
Looked into it and it should not be a concern, since we eventually call `version::resolve` to obtain the right version to use directly in the `build_chan_open_try` method.